cmd/initializer/internal/config_generator.go

```diff
@@ -34,7 +34,11 @@ type Options struct {
 	Namespace      string
 	ComponentKind  string
 
-	// For generating config of datanode.
+	// For generating config of datanode or flownode.
+	RPCPort     int32
```

Comment:
question: Seem naming `FlownodeRPCPort` and `FlownodeServiceName` better ?

Reply:
I don't think so. The `DatanodeRPCPort` and `DatanodeServiceName` will be removed in the future. No need to name a `Flownode*`.